package.json

```diff
@@ -6,7 +6,8 @@
   "repository": "https://github.com/AtomLinter/linter-reek.git",
   "license": "MIT",
   "engines": {
-    "atom": ">=1.0.0 <2.0.0"
+    "atom": ">=1.0.0 <2.0.0",
+    "node": ">=4.x.x"
```

Comment:
This isn't necessary. The problem was that Atom versions less than v1.10.0 bundled Node.js v0.10.40 with `apm`... and set the `$PATH` to use that version of `node` before any others in the build script.

(Atom itself has been on later versions of Node.js than this for a while.